00000100:00000400:0.0F:1574201492.593276:0:4834:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has timed out for slow reply: [sent 1574201485/real 1574201485]  req@ffff96004a888900 x1650385116024144/t0(0) o400->lustre-MDT0000-mdc-ffff9600bba42800@10.24.85.134@tcp:12/10 lens 224/224 e 0 to 1 dl 1574201492 ref 1 fl Rpc:XN/0/ffffffff rc 0/-1
00000100:02000400:0.0:1574201492.593314:0:4834:0:(import.c:179:ptlrpc_set_import_discon()) lustre-MDT0000-mdc-ffff9600bba42800: Connection to lustre-MDT0000 (at 10.24.85.134@tcp) was lost; in progress operations using this service will wait for recovery to complete
00000100:00080000:0.0:1574201492.593336:0:4834:0:(import.c:188:ptlrpc_set_import_discon()) ffff960063dfb800 lustre-MDT0000_UUID: changing import state from FULL to DISCONN
00000100:00080000:0.0:1574201492.593341:0:4834:0:(import.c:418:ptlrpc_pinger_force()) lustre-MDT0000_UUID: waking up pinger s:DISCONN
00000100:00080000:0.0:1574201492.593370:0:4835:0:(pinger.c:217:ptlrpc_pinger_process_import()) e06734de-5238-64b2-8ad8-e120574318e5->lustre-MDT0000_UUID: level DISCONN/3 force 1 force_next 0 deactive 0 pingable 1 suppress 0
00000100:00080000:0.0:1574201492.593376:0:4835:0:(recover.c:58:ptlrpc_initiate_recovery()) lustre-MDT0000_UUID: starting recovery
00000100:00080000:0.0:1574201492.593378:0:4835:0:(import.c:681:ptlrpc_connect_import()) ffff960063dfb800 lustre-MDT0000_UUID: changing import state from DISCONN to CONNECTING
00000100:00080000:0.0:1574201492.593384:0:4835:0:(import.c:524:import_select_connection()) lustre-MDT0000-mdc-ffff9600bba42800: connect to NID 10.24.85.134@tcp last attempt 4565257432
00000100:00080000:0.0:1574201492.593392:0:4835:0:(import.c:602:import_select_connection()) lustre-MDT0000-mdc-ffff9600bba42800: import ffff960063dfb800 using connection 10.24.85.134@tcp/10.24.85.134@tcp
00000100:00000400:0.0:1574201498.592995:0:4832:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has timed out for slow reply: [sent 1574201492/real 1574201492]  req@ffff96004a888900 x1650385116024208/t0(0) o38->lustre-MDT0000-mdc-ffff9600bba42800@10.24.85.134@tcp:12/10 lens 520/544 e 0 to 1 dl 1574201498 ref 1 fl Rpc:XN/0/ffffffff rc 0/-1
00000100:00080000:0.0:1574201498.593030:0:4832:0:(import.c:1284:ptlrpc_connect_interpret()) ffff960063dfb800 lustre-MDT0000_UUID: changing import state from CONNECTING to DISCONN
00000100:00080000:0.0:1574201498.593035:0:4832:0:(import.c:1330:ptlrpc_connect_interpret()) recovery of lustre-MDT0000_UUID on 10.24.85.134@tcp failed (-110)
00000100:00080000:0.0:1574201517.592897:0:4835:0:(pinger.c:217:ptlrpc_pinger_process_import()) e06734de-5238-64b2-8ad8-e120574318e5->lustre-MDT0000_UUID: level DISCONN/3 force 0 force_next 0 deactive 0 pingable 1 suppress 1
00000100:00080000:0.0:1574201517.592907:0:4835:0:(recover.c:58:ptlrpc_initiate_recovery()) lustre-MDT0000_UUID: starting recovery
00000100:00080000:0.0:1574201517.592909:0:4835:0:(import.c:681:ptlrpc_connect_import()) ffff960063dfb800 lustre-MDT0000_UUID: changing import state from DISCONN to CONNECTING
00000100:00080000:0.0:1574201517.592918:0:4835:0:(import.c:524:import_select_connection()) lustre-MDT0000-mdc-ffff9600bba42800: connect to NID 10.24.85.134@tcp last attempt 4568188510
00000100:00080000:0.0:1574201517.592922:0:4835:0:(import.c:568:import_select_connection()) lustre-MDT0000-mdc-ffff9600bba42800: tried all connections, increasing latency to 6s
00000100:00080000:0.0:1574201517.592929:0:4835:0:(import.c:602:import_select_connection()) lustre-MDT0000-mdc-ffff9600bba42800: import ffff960063dfb800 using connection 10.24.85.134@tcp/10.24.85.134@tcp
00000800:00000100:0.0:1574201527.254758:0:4828:0:(socklnd_cb.c:2282:ksocknal_find_timed_out_conn()) A connection with 12345-10.29.161.60@tcp (10.29.161.60:988) timed out; the network or node may be down.
00000100:00000400:0.0:1574201528.609614:0:4832:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has timed out for slow reply: [sent 1574201517/real 1574201517]  req@ffff96004a888c00 x1650385116024240/t0(0) o38->lustre-MDT0000-mdc-ffff9600bba42800@10.24.85.134@tcp:12/10 lens 520/544 e 0 to 1 dl 1574201528 ref 1 fl Rpc:XN/0/ffffffff rc 0/-1
00000100:00080000:0.0:1574201528.609673:0:4832:0:(import.c:1284:ptlrpc_connect_interpret()) ffff960063dfb800 lustre-MDT0000_UUID: changing import state from CONNECTING to DISCONN
00000100:00080000:0.0:1574201528.609681:0:4832:0:(import.c:1330:ptlrpc_connect_interpret()) recovery of lustre-MDT0000_UUID on 10.24.85.134@tcp failed (-110)
00000800:00000100:0.0:1574201536.254572:0:4828:0:(socklnd_cb.c:2323:ksocknal_find_timed_out_conn()) Timeout sending data to 12345-10.24.85.134@tcp (10.24.85.134:988) the network or that node may be down.
00000100:00080000:0.0:1574201542.592944:0:4835:0:(pinger.c:217:ptlrpc_pinger_process_import()) e06734de-5238-64b2-8ad8-e120574318e5->lustre-MDT0000_UUID: level DISCONN/3 force 0 force_next 0 deactive 0 pingable 1 suppress 1
00000100:00080000:0.0:1574201542.592953:0:4835:0:(recover.c:58:ptlrpc_initiate_recovery()) lustre-MDT0000_UUID: starting recovery
00000100:00080000:0.0:1574201542.592956:0:4835:0:(import.c:681:ptlrpc_connect_import()) ffff960063dfb800 lustre-MDT0000_UUID: changing import state from DISCONN to CONNECTING
00000100:00080000:0.0:1574201542.592965:0:4835:0:(import.c:524:import_select_connection()) lustre-MDT0000-mdc-ffff9600bba42800: connect to NID 10.24.85.134@tcp last attempt 4568213510
00000100:00080000:0.0:1574201542.592968:0:4835:0:(import.c:568:import_select_connection()) lustre-MDT0000-mdc-ffff9600bba42800: tried all connections, increasing latency to 11s
00000100:00080000:0.0:1574201542.592975:0:4835:0:(import.c:602:import_select_connection()) lustre-MDT0000-mdc-ffff9600bba42800: import ffff960063dfb800 using connection 10.24.85.134@tcp/10.24.85.134@tcp
00000400:00000100:0.0:1574201545.306913:0:4827:0:(lib-socket.c:645:lnet_sock_connect()) Error -113 connecting 10.24.86.108/1020 -> 10.24.85.134/988
00000400:00000100:0.0:1574201545.306942:0:4827:0:(acceptor.c:112:lnet_connect_console_error()) Connection to 10.24.85.134@tcp at host 10.24.85.134 was unreachable: the network or that node may be down, or Lustre may be misconfigured.
00000800:00000100:0.0:1574201545.306956:0:4827:0:(socklnd_cb.c:425:ksocknal_txlist_done()) Deleting packet type 1 len 520 10.24.86.108@tcp->10.24.85.134@tcp
00000100:00000400:0.0:1574201545.306989:0:4832:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has failed due to network error: [sent 1574201542/real 1574201545]  req@ffff96004a889b00 x1650385116024320/t0(0) o38->lustre-MDT0000-mdc-ffff9600bba42800@10.24.85.134@tcp:12/10 lens 520/544 e 0 to 1 dl 1574201558 ref 1 fl Rpc:eXN/0/ffffffff rc 0/-1
00000100:00080000:0.0:1574201545.307015:0:4832:0:(import.c:1284:ptlrpc_connect_interpret()) ffff960063dfb800 lustre-MDT0000_UUID: changing import state from CONNECTING to DISCONN
00000100:00080000:0.0:1574201545.307019:0:4832:0:(import.c:1330:ptlrpc_connect_interpret()) recovery of lustre-MDT0000_UUID on 10.24.85.134@tcp failed (-110)
00000100:00000400:0.0:1574201565.595769:0:4833:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has timed out for slow reply: [sent 1574201517/real 1574201517]  req@ffff96004a888000 x1650385116024288/t0(0) o400->lustre-OST0002-osc-ffff9600bba42800@10.29.161.60@tcp:28/4 lens 224/224 e 0 to 1 dl 1574201565 ref 1 fl Rpc:XN/0/ffffffff rc 0/-1
00000100:02000400:0.0:1574201565.595804:0:4833:0:(import.c:179:ptlrpc_set_import_discon()) lustre-OST0002-osc-ffff9600bba42800: Connection to lustre-OST0002 (at 10.29.161.60@tcp) was lost; in progress operations using this service will wait for recovery to complete
00000100:00080000:0.0:1574201565.595819:0:4833:0:(import.c:188:ptlrpc_set_import_discon()) ffff9600b9577800 lustre-OST0002_UUID: changing import state from FULL to DISCONN
00000100:00080000:0.0:1574201565.595824:0:4833:0:(import.c:418:ptlrpc_pinger_force()) lustre-OST0002_UUID: waking up pinger s:DISCONN
00000100:00080000:0.0:1574201565.595854:0:4835:0:(pinger.c:217:ptlrpc_pinger_process_import()) e06734de-5238-64b2-8ad8-e120574318e5->lustre-OST0002_UUID: level DISCONN/3 force 1 force_next 0 deactive 0 pingable 1 suppress 0
00000100:00080000:0.0:1574201565.595860:0:4835:0:(recover.c:58:ptlrpc_initiate_recovery()) lustre-OST0002_UUID: starting recovery
00000100:00080000:0.0:1574201565.595863:0:4835:0:(import.c:681:ptlrpc_connect_import()) ffff9600b9577800 lustre-OST0002_UUID: changing import state from DISCONN to CONNECTING
00000100:00080000:0.0:1574201565.595872:0:4835:0:(import.c:524:import_select_connection()) lustre-OST0002-osc-ffff9600bba42800: connect to NID 10.29.161.60@tcp last attempt 4567831492
00000100:00080000:0.0:1574201565.595881:0:4835:0:(import.c:602:import_select_connection()) lustre-OST0002-osc-ffff9600bba42800: import ffff9600b9577800 using connection 10.29.161.60@tcp/10.29.161.60@tcp
00000100:00000400:0.0:1574201575.595683:0:4832:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has timed out for sent delay: [sent 1574201565/real 0]  req@ffff96004a888000 x1650385116024384/t0(0) o8->lustre-OST0002-osc-ffff9600bba42800@10.29.161.60@tcp:28/4 lens 520/544 e 0 to 1 dl 1574201575 ref 2 fl Rpc:XN/0/ffffffff rc 0/-1
00000100:00080000:0.0:1574201590.596133:0:4835:0:(pinger.c:217:ptlrpc_pinger_process_import()) e06734de-5238-64b2-8ad8-e120574318e5->lustre-MDT0000_UUID: level DISCONN/3 force 0 force_next 0 deactive 0 pingable 1 suppress 1
00000100:00080000:0.0:1574201590.596143:0:4835:0:(recover.c:58:ptlrpc_initiate_recovery()) lustre-MDT0000_UUID: starting recovery
00000100:00080000:0.0:1574201590.596145:0:4835:0:(import.c:681:ptlrpc_connect_import()) ffff960063dfb800 lustre-MDT0000_UUID: changing import state from DISCONN to CONNECTING
00000100:00080000:0.0:1574201590.596154:0:4835:0:(import.c:524:import_select_connection()) lustre-MDT0000-mdc-ffff9600bba42800: connect to NID 10.24.85.134@tcp last attempt 4568238510
00000100:00080000:0.0:1574201590.596158:0:4835:0:(import.c:568:import_select_connection()) lustre-MDT0000-mdc-ffff9600bba42800: tried all connections, increasing latency to 16s
00000100:00080000:0.0:1574201590.596165:0:4835:0:(import.c:602:import_select_connection()) lustre-MDT0000-mdc-ffff9600bba42800: import ffff960063dfb800 using connection 10.24.85.134@tcp/10.24.85.134@tcp
00000100:00080000:0.0:1574201590.596184:0:4835:0:(pinger.c:217:ptlrpc_pinger_process_import()) e06734de-5238-64b2-8ad8-e120574318e5->lustre-OST0002_UUID: level CONNECTING/4 force 0 force_next 0 deactive 0 pingable 1 suppress 1
00000100:00080000:0.0:1574201590.596189:0:4835:0:(pinger.c:230:ptlrpc_pinger_process_import()) e06734de-5238-64b2-8ad8-e120574318e5->lustre-OST0002_UUID: not pinging (in recovery or recovery disabled: CONNECTING)
00000100:00000400:0.0:1574201591.596068:0:4833:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has timed out for sent delay: [sent 1574201542/real 0]  req@ffff96004a889200 x1650385116024368/t0(0) o400->lustre-OST0002-osc-ffff9600bba42800@10.29.161.60@tcp:28/4 lens 224/224 e 0 to 1 dl 1574201590 ref 2 fl Rpc:XN/0/ffffffff rc 0/-1
00000100:00080000:0.0:1574201591.596098:0:4833:0:(import.c:203:ptlrpc_set_import_discon()) osc: import ffff9600b9577800 already not connected (conn 45, was 44): CONNECTING
00000400:00000100:0.0:1574201593.411129:0:4826:0:(lib-socket.c:645:lnet_sock_connect()) Error -113 connecting 10.24.86.108/1023 -> 10.24.85.134/988
00000400:00000100:0.0:1574201593.411156:0:4826:0:(acceptor.c:112:lnet_connect_console_error()) Connection to 10.24.85.134@tcp at host 10.24.85.134 was unreachable: the network or that node may be down, or Lustre may be misconfigured.
00000800:00000100:0.0:1574201593.411170:0:4826:0:(socklnd_cb.c:425:ksocknal_txlist_done()) Deleting packet type 1 len 520 10.24.86.108@tcp->10.24.85.134@tcp
00000100:00000400:0.0:1574201593.411201:0:4832:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has failed due to network error: [sent 1574201590/real 1574201593]  req@ffff96004a888900 x1650385116024416/t0(0) o38->lustre-MDT0000-mdc-ffff9600bba42800@10.24.85.134@tcp:12/10 lens 520/544 e 0 to 1 dl 1574201611 ref 1 fl Rpc:eXN/0/ffffffff rc 0/-1
00000100:00080000:0.0:1574201593.411226:0:4832:0:(import.c:1284:ptlrpc_connect_interpret()) ffff960063dfb800 lustre-MDT0000_UUID: changing import state from CONNECTING to DISCONN
00000100:00080000:0.0:1574201593.411230:0:4832:0:(import.c:1330:ptlrpc_connect_interpret()) recovery of lustre-MDT0000_UUID on 10.24.85.134@tcp failed (-110)
00000800:00000100:0.0:1574201603.254593:0:4828:0:(socklnd_cb.c:425:ksocknal_txlist_done()) Deleting packet type 1 len 224 10.24.86.108@tcp->10.29.161.60@tcp
00000100:00080000:0.0:1574201615.595558:0:4835:0:(pinger.c:217:ptlrpc_pinger_process_import()) e06734de-5238-64b2-8ad8-e120574318e5->lustre-MDT0000_UUID: level DISCONN/3 force 0 force_next 0 deactive 0 pingable 1 suppress 1
00000100:00080000:0.0:1574201615.595568:0:4835:0:(recover.c:58:ptlrpc_initiate_recovery()) lustre-MDT0000_UUID: starting recovery
00000100:00080000:0.0:1574201615.595570:0:4835:0:(import.c:681:ptlrpc_connect_import()) ffff960063dfb800 lustre-MDT0000_UUID: changing import state from DISCONN to CONNECTING
00000100:00080000:0.0:1574201615.595579:0:4835:0:(import.c:524:import_select_connection()) lustre-MDT0000-mdc-ffff9600bba42800: connect to NID 10.24.85.134@tcp last attempt 4568286513
00000100:00080000:0.0:1574201615.595583:0:4835:0:(import.c:568:import_select_connection()) lustre-MDT0000-mdc-ffff9600bba42800: tried all connections, increasing latency to 21s
00000100:00080000:0.0:1574201615.595607:0:4835:0:(import.c:602:import_select_connection()) lustre-MDT0000-mdc-ffff9600bba42800: import ffff960063dfb800 using connection 10.24.85.134@tcp/10.24.85.134@tcp
00000100:00080000:0.0:1574201615.595628:0:4835:0:(pinger.c:217:ptlrpc_pinger_process_import()) e06734de-5238-64b2-8ad8-e120574318e5->lustre-OST0002_UUID: level CONNECTING/4 force 0 force_next 0 deactive 0 pingable 1 suppress 1
00000100:00080000:0.0:1574201615.595633:0:4835:0:(pinger.c:230:ptlrpc_pinger_process_import()) e06734de-5238-64b2-8ad8-e120574318e5->lustre-OST0002_UUID: not pinging (in recovery or recovery disabled: CONNECTING)
00000400:00000100:0.0:1574201617.466891:0:4825:0:(lib-socket.c:645:lnet_sock_connect()) Error -113 connecting 10.24.86.108/1023 -> 10.24.85.134/988
00000400:00000100:0.0:1574201617.466918:0:4825:0:(acceptor.c:112:lnet_connect_console_error()) Connection to 10.24.85.134@tcp at host 10.24.85.134 was unreachable: the network or that node may be down, or Lustre may be misconfigured.
00000800:00000100:0.0:1574201617.466932:0:4825:0:(socklnd_cb.c:425:ksocknal_txlist_done()) Deleting packet type 1 len 520 10.24.86.108@tcp->10.24.85.134@tcp
00000100:00000400:0.0:1574201617.466967:0:4832:0:(client.c:2116:ptlrpc_expire_one_request()) @@@ Request sent has failed due to network error: [sent 1574201615/real 1574201617]  req@ffff96004a888900 x1650385116024480/t0(0) o38->lustre-MDT0000-mdc-ffff9600bba42800@10.24.85.134@tcp:12/10 lens 520/544 e 0 to 1 dl 1574201641 ref 1 fl Rpc:eXN/0/ffffffff rc 0/-1
00000100:00080000:0.0:1574201617.466992:0:4832:0:(import.c:1284:ptlrpc_connect_interpret()) ffff960063dfb800 lustre-MDT0000_UUID: changing import state from CONNECTING to DISCONN
00000100:00080000:0.0:1574201617.466996:0:4832:0:(import.c:1330:ptlrpc_connect_interpret()) recovery of lustre-MDT0000_UUID on 10.24.85.134@tcp failed (-110)
00000800:00000100:0.0:1574201628.254447:0:4828:0:(socklnd_cb.c:425:ksocknal_txlist_done()) Deleting packet type 1 len 520 10.24.86.108@tcp->10.29.161.60@tcp
00000100:00080000:0.0:1574201628.254467:0:4832:0:(import.c:1284:ptlrpc_connect_interpret()) ffff9600b9577800 lustre-OST0002_UUID: changing import state from CONNECTING to DISCONN
00000100:00080000:0.0:1574201628.254470:0:4832:0:(import.c:1330:ptlrpc_connect_interpret()) recovery of lustre-OST0002_UUID on 10.29.161.60@tcp failed (-110)
